<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NGQ Builds Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="wInQueueBuilds" native="true">
      <layout class="QVBoxLayout" name="vlInQueueBuilds">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="wInBuildingBuilds" native="true">
      <layout class="QVBoxLayout" name="vlInBuildingBuilds">
       <property name="margin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>461</width>
         <height>355</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vlBuildsContainer"/>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbBookBuild">
      <property name="text">
       <string>Book build</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lManagerStatus">
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionConnection"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionUpdate"/>
   </widget>
   <addaction name="menuActions"/>
   <addaction name="menuSettings"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConnection">
   <property name="text">
    <string>Connection</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>Update</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
